git.go

```diff
@@ -79,29 +79,30 @@ func (r *BumperRepo) getModifiedFiles(last bool) ([]string, error) {
 		return nil, err
 	}
 
-	parentCommit, err := headCommit.Parent(0)
+	commits, err := r.git.Log(&git.LogOptions{From: headRef.Hash()})
 	if err != nil {
 		return nil, err
 	}
 
+	var prevCommit *object.Commit
 	commitsNum := 0
-	for {
-		if commitsNum > 0 {
-			headCommit = parentCommit
-		}
 
-		headTree, _ := headCommit.Tree()
+	err = commits.ForEach(func(commit *object.Commit) error {
+		commitsNum++
 
-		parentCommit, err = headCommit.Parent(0)
-		if err != nil {
-			return nil, err
+		// Process last commit.
+		if prevCommit == nil {
+			prevCommit = commit
+			return nil
 		}
 
-		parentTree, _ := parentCommit.Tree()
+		// Process parents.
```

Comment:
Are you sure `parents` is the right wording in this context ? It just look like previous commit to me

Reply:
For me it's the same) but  previous commit - yes, better. Changed.